### PR TITLE
Cais gateway UI changes

### PR DESCRIPTION
**Pull request by CaisManai**
Revamped the UI and fixed several UI bugs. Only tested on desktop.

### CODE REVIEW

File: tools/walletextension/api/staticOG/index.html [hunk 1]
```diff
@@ -11,25 +11,58 @@
...
+    <h2>Welcome to the Obscuro Gateway</h2>
+    <form>
+        <div id="information">A 3 click process to setup encrypted communication between MetaMask and the Obscuro Network.
+            Upon hitting Begin, you'll be asked to:
+            <ol>
+                <li>
+                    Add the Obscuro Testnet network as a new network to MetaMask. Check the Network URL base is https://testnet.obscu.ro and the chain ID is 443 and hit approve  
+                </li>
+                <li>
+                    Allow MetaMask to switch networks to the Obscuro Testnet network
```
Comment (zkokelj) [inline tools/walletextension/api/staticOG/index.html:30]: We should probably mention that users need to select which accounts they want to connect? (for the first time users)

File: tools/walletextension/api/staticOG/javascript.js [hunk 2]
```diff
@@ -271,22 +275,35 @@ const initialize = async () => {
...
 
     function displayOnlyJoin() {
         joinButton.style.display = "block"
         revokeUserIDButton.style.display = "none"
         accountsTable.style.display = "none"
+        informationArea.style.display = "block"
+        informationArea2.style.display = "none"
+
+        beginBox.style.visibility = "visible";
+        spinner.style.visibility = "hidden";
```
Comment (zkokelj) [inline tools/walletextension/api/staticOG/javascript.js:297]: When I add accounts and refresh page it only shows us the spinner

File: tools/walletextension/api/staticOG/javascript.js
Comment (0xjba) [inline tools/walletextension/api/staticOG/javascript.js:1]: Please revert back to the Hard coded gateway address & any usage of it.
Line 1, 14-18, Line 34

**Review by 0xjba — CHANGES_REQUESTED** [review]
Please revert back the Hard coded gateway address & any usage of it.
Line 1, 14-18, Line 34

File: tools/walletextension/api/staticOG/style.css [hunk 3]
```diff
@@ -35,11 +203,13 @@ input { display: table-cell; }
 
 /* basic style for the accounts table*/
 table {
```
Comment (zkokelj) [inline tools/walletextension/api/staticOG/style.css:205]: Maybe center the table and use same style as the rest of the webpage:

image

In the table we display all the accounts that are connected to metamask (for this URL), even if they are not connected to Obscuro (connected = False), but we don't give an option to the user to add them. Is this ok?

**Review by zkokelj — COMMENTED** [review]
Nice improvements! 

Mentioned a few things in the comments, but nothing major (not working after refresh is the biggest issue in my opinion).

File: tools/walletextension/api/staticOG/style.css (hunk 3 above)
Comment (CaisManai) [reply tools/walletextension/api/staticOG/style.css:205]: It's not ideal, but it's okay for this first version. I'll raise a ticket for it.

**Pull request merged**